### PR TITLE
Create Project Proposal Draft2.md

### PR DESCRIPTION
**Pull request by DJ-Moye**
Update to the Project Proposal Draft so far for Team 1 who is over the IEEE SECON Hardware Competition.

### CODE REVIEW

File: Project Proposal Draft2.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+  - Alex Cruz – Electrical Engineering major. Proficient in Python and C++. Some familiarity with machine learning.
+  - Sam Hunter – Computer Engineering major. Proficient in C++.
+  - Dakota Moye – Electrical Engineering major with minors in Physics and Math. Proficient in C++, MATLAB, Arduino IDE. Some experience with Python, Inventor, 3D Printing, PuTTY.
+  - Alejandro Moore – Electrical Engineering major. Proficient in C++, C#, C, Assembly, Python, AutoCAD, PLC, and Altium. PCB Design experience and BNC/Ethernet cable installation experience.
+- Contributions
+  - Sean Borchers – Motor Control
+  - Alex Cruz - Intro, Navigation and Master Control, Measures of Success
+  - Sam Hunter - Object and Line Detection
+  - Dakota Moye – Sensor for Navigation and Location
+  - Alejandro Moore -
```
[the remark this answers is not in the document]
Comment (PapiShuay) [reply Project Proposal Draft2.md]: Contributions added

File: Project Proposal Draft2.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+|Brushless Motors|~$60.00|4|$240.00|
+|Microcontroller|~$30.00|2|$60.00|
+|Motor Drivers|~$20.00|2|$40.00|
+|Intel® RealSense™ Depth Camera D415</h1>|~$272.00|1|$272.00|
+|Motor Encoders|~$23.00|1|$23.00|
+|Total||||
+
+Timeline
+
+ ![image](https://github.com/ACruz-42/F24_Team1_CapstoneDemo/blob/3b0272b824db69aba49f735e246663ff42dbf47a/Reports/Photos/Project%20Proposal/Gantt_Draft2.png)
```
[the remark this answers is not in the document]
Comment (PapiShuay) [reply Project Proposal Draft2.md]: Will do!

File: Project Proposal Draft2.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+5)	The robot shall stay on the game field. Rule S01 [2].
+6)	The robot may have a single, simple electrical start button that shall be clearly labelled. The operation of this button shall prevent the robot from moving for five seconds. Rule R03 [1].
+7)	The robot shall have a clearly labelled emergency stop button that immediately halts all functions safely and quickly. Rule R05 [1].
+8)	The robot shall have a source of identification clearly associating it with Tennessee Technological University. Rule R10 [1].
+9)	The robot may use light sources that shall not strobe, be excessively bright, or pose safety issues. Rule R17 [1].
+10)	The robot shall cease operation of all its units after the allotted 3-minute timeframe of play is over, and no game element positions can be disturbed after time expires. Rule G07 [2]. 
+11)	The robot shall account for background interference in the competition environment. Rule R13 [1].
+12)	The robot shall sustain power of 30V maximum. Rule R16 [1].
+
+### Emergency Stop
```
[the remark this answers is not in the document]
Comment (PapiShuay) [reply Project Proposal Draft2.md]: E Stop subsection moved to Problem Section.

File: Project Proposal Draft2.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
...otor driver. The driver also provides voltage regulation for the higher threshold of the motor, precision control for speed and torque, overload protection, and noise protection for the circuits involving low-power logic control [6]. Regarding the motors themselves, the team that worked on the previous robot base used DC motors due to their increased control capabilities and starting torque in comparison to AC motors. However, they were concerned with the back electromotive force that moves in the opposite direction of the motor spin. The back emf can interfere with low-power, highly sensitive logic circuits. This problem is also present when the motor stops or quickly slows down [7]. During the match, the robot will be required to quickly move across the area of play and collect astral material automatically. The robot will need to stop and start again numerous times as it makes its way around the field. The back emf generated by continual stopping and starting of the motors would be harmful to the control circuitry, generating noise. It is critical to reduce this emf noise generated by the motor as much as possible for optimal communication between the controllers and mechanisms.
```
[the remark this answers is not in the document]
Comment (SeanBorchers) [reply Project Proposal Draft2.md]: Removed the first section of the second paragraph discussing motor component definitions. Will save these details for design. Also created a more concise survey of solutions for Motor Control using the details and sources that were most relevant and applicable to the project.

File: Project Proposal Draft2.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+|Accelerometer and Gyroscope|~$40|1|$40|
+|Lithium Iron Phosphate Battery|$91.43|1|$91.43|
+|Lithium Iron Phosphate Battery Charger|$129.27|1|$129.27|
+|Navigation & Master Control|~$250|||
+|Brushless Motors|~$60.00|4|$240.00|
+|Microcontroller|~$30.00|2|$60.00|
+|Motor Drivers|~$20.00|2|$40.00|
+|Intel® RealSense™ Depth Camera D415</h1>|~$272.00|1|$272.00|
+|Motor Encoders|~$23.00|1|$23.00|
+|Total||||
```
[the remark this answers is not in the document]
Comment (SeanBorchers) [reply Project Proposal Draft2.md]: Added the total based on the compiled list of components: $1169.70

File: Project Proposal Draft2.md [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+
+ ![image](https://github.com/ACruz-42/F24_Team1_CapstoneDemo/blob/3b0272b824db69aba49f735e246663ff42dbf47a/Reports/Photos/Project%20Proposal/Gantt_Draft2.png)
+
+- Personnel
+  - Sean Borchers - Electrical Engineering major with a Mechatronics concentration. Proficient in CAD software (AutoCAD, Inventor, SolidWorks), MATLAB, C++
+  - Alex Cruz – Electrical Engineering major. Proficient in Python and C++. Some familiarity with machine learning.
+  - Sam Hunter – Computer Engineering major. Proficient in C++.
+  - Dakota Moye – Electrical Engineering major with minors in Physics and Math. Proficient in C++, MATLAB, Arduino IDE. Some experience with Python, Inventor, 3D Printing, PuTTY.
+  - Alejandro Moore – Electrical Engineering major. Proficient in C++, C#, C, Assembly, Python, AutoCAD, PLC, and Altium. PCB Design experience and BNC/Ethernet cable installation experience.
+- Contributions
```
[the remark this answers is not in the document]
Comment (DJ-Moye) [reply Project Proposal Draft2.md:169]: Contributions are slightly below. They are specific to this document. Personnel lists our skills.

**Review by micahr234 — APPROVED** [review]

**Pull request merged**